data/mods/Dark-Skies-Above/DESIGN.md

```diff
@@ -2,42 +2,41 @@
 
-In short, Dark Skies is a science fiction/science fantasy story focused around an ongoing alien invasion and the survival of those left in one of the hot-sites: the New England Exclusion Zone.
+Edit: You can also contact me, '@Zlorthishen'.  I have a few ideas for directions for this mod which aren't listed in this file. Expect a free season pass DLC named "Peach Trees" soon™.
 
 
-# Contributing
-
-Since Dark Skies is distributed with the base game, contributions are welcome. If you have any questions, you can contact me on Discord via `@ephemeral_storyteller` through the official developer or official community severs.
+# Summary of Intent and Content
 
+Mechanically, Dark Skies Above began as a project to demonstrate how you can push the game engine to present different stories. Unlike the base game, a narrative arc is assumed: the player will want find a way to make occupation of Earth too costly for the aliens to continue, or die trying. This is, of course, by no means an "easy" or "short" process, nor is it necessarily a requirement. This is done largely to compensate for the fact that Dark Skies does not play well with other mods or some aspects of active development, thus needing a tighter experience.
 
-# Content
+In short, Dark Skies is a science fiction/science fantasy story focused around an ongoing alien invasion and the survival of those left in one of the hot-sites: the New England Exclusion Zone.
 
 Summarized below are the rough design directions currently intended.
 
 
 ## The World
 
-The Earth presented in Dark Skies is a little closer to our current tech level (things you'd reasonably find in the 21st century). It is still roughly "some time in the near future", and some of the advancements in technology reflect this. The effects of global warming are already starting to increase the severity and frequency of natural disasters, while tensions between trade leaders are starting to grow due to global economic depression. As if things couldn't get any worse, astronomical institutions across the globe one day detected a peculiar gravitational anomaly just outside the Moon's orbit, initially considered a quaint discovery before alien spacecraft begin to spill from it...
+The Earth presented in Dark Skies is a little closer to our current tech level (things you'd reasonably find in the 21st century). It is still roughly "some time in the near future", and some of the advancements in technology reflect this. The effects of global warming are already starting to increase the severity and frequency of natural disasters, COVID-21 rages unchecked while tensions between trade leaders are starting to grow due to global economic depression. As if things couldn't get any worse, astronomical institutions across the globe one day detected a peculiar gravitational anomaly just outside the Moon's orbit, initially considered a quaint discovery before alien spacecraft begin to spill from it...
```

Comment:
The design document mentions Covid-21 out of nowhere, and I dunno the thoughts on mentioning current day things like that which can get incredibly dated quickly.

Reply:
<adds back in snippets for Presidential Election Fliers>